src/components/Datepicker/Datepicker.js

```diff
@@ -354,15 +355,27 @@ export class Datepicker extends React.Component {
 
     const onClick = () => inRange && this.onDatepickerChange(day);
 
+    const isToday = day.isSame(moment(), 'day');
+
+    const isSelected = day.isSame(moment(inputValue, defaultDateFormat), 'day');
+
     const classes = {
       'slds-is-today': day.isSame(moment(), 'day'),
       'slds-disabled-text': !inRange,
-      'slds-is-selected': day.isSame(moment(inputValue, defaultDateFormat), 'day'),
+      'slds-is-selected': isSelected,
     };
 
     return (
-      <td key={dayIndex} className={cx(classes)} headers={day.day()}>
+      <td
+        aria-disabled={!inRange ? 'true' : null}
```

Comment:
A simple `aria-disabled={!inRange}` should suffice. In the end, we just want to make sure the attribute is set when `inRange` is falsy.

Reply:
I just added the null option because in the lds code examples, the aria-disabled attribute does not exist if inRange is truthy. When I set it to false, I get an html attribute, with null, react omits the attribute...